conans/test/integration/devflow_test.py

```diff
@@ -244,7 +244,7 @@ def build_local_different_folders_test(self):
         install_dir = os.path.join(client.current_folder, "install_x86_64")
         tools.replace_in_file(os.path.join(client.current_folder, "src", "hello", "CMakeLists.txt"),
                               "${CMAKE_BINARY_DIR}/conanbuildinfo.cmake",
-                              '"%s/conanbuildinfo.cmake"' % install_dir,
```

Comment:
why necessary? It is not failing so far

Reply:
I was wondering the same and I have no idea...

Reply:
It failed in [this CI job](https://conan-ci.jfrog.info/blue/organizations/jenkins/ConanTestSuite/detail/PR-5069/2/pipeline), I would want to know what's happening, but if it is needed, this PR is ok for me and ready to merge.